src/components/ClearRefinements/__tests__/ClearRefinements-test.js

```diff
@@ -20,16 +20,16 @@ describe('ClearRefinements', () => {
   };
 
   it('should render <ClearRefinements />', () => {
-    const tree = renderer
-      .create(<ClearRefinements {...defaultProps} />)
-      .toJSON();
-    expect(tree).toMatchSnapshot();
+    const wrapper = mount(<ClearRefinements {...defaultProps} />);
+
+    expect(wrapper).toMatchSnapshot();
```

Comment:
I might have missed it but I'm surprised the snapshot didn't change at all after this. 🤔 
I guess the build would have failed if they didn't match

Reply:
Emzyme's `mount` is a bit more explicit than React Test Renderer but in this case they output the same component.